db/__tests__/seed.test.js

```diff
@@ -112,5 +111,63 @@ test('GET 200: responds with array of article objects in descending order', () =
         })
     })
 })
+test('GET 200: responds with array of article objects in descending order', () => {
+    return request(app)
+    .get('/api/articles')
+    .expect(200)
+    .then(({ body }) => {
+        const { articles } = body
+        expect(articles).toBeSortedBy('created_at',{ descending: true })
+    })
+})
+})
+
+describe('/api/articles/:article_id/comments', () => {
+test('GET 200: responds with an array of comments for the given article_id', () => {
+    return request(app)
+    .get('/api/articles/1/comments')
+    .expect(200)
+    .then(({ body }) => {
+    const { comments } = body
+    expect(comments.length).toBe(11)
+    comments.forEach((comment) => {
+        expect(comment).toMatchObject(expect.objectContaining({
+            comment_id: expect.any(Number),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            article_id: expect.any(Number),
```

Comment:
We can be more specific with the article_id as we know what article we have requested